### PR TITLE
embed adaptive Quantization method

### PR DESCRIPTION
**Pull request by Jamaika1**

### CODE REVIEW

File: src_base/xeve_bsw.c [hunk 1]
```diff
@@ -198,7 +198,7 @@ int xeve_bsw_write(XEVE_BSW * bs, u32 val, int len) /* len(1 ~ 32) */
     int leftbits;
 
     xeve_assert(bs);
-    xeve_assert(len <= 32); // to avoid shifting by a negative value
+    //xeve_assert(len <= 32); // to avoid shifting by a negative value
```
Comment (dariusz-f) [inline src_base/xeve_bsw.c:201]: Describe why this change is required?

Comment (Jamaika1) [reply src_base/xeve_bsw.c:201]: If adaptive quantization method functions are enabled then I receive error.

Comment (dariusz-f) [reply src_base/xeve_bsw.c:201]: Proper way is to solve issue not disable assert.

Comment (dariusz-f) [reply src_base/xeve_bsw.c]: Closing change as invalid.
If you have problems with adaptive quantization please open issue with description and data that allow to find root cause of problem.
